applications/nightly_build/test_transformers.py

```diff
@@ -46,11 +48,16 @@ def _get_tokenzier(self, tokenizer_file):
 
     def _prepare_inputs(self, tokenizer):
         raw_data = json.dumps({
-            'text': 'The quick brown fox jumps over the lazy dog.'
+            'text': 'The quick brown fox jumps over lazy dog.'
         })
         text = json.loads(raw_data)['text']
-        inputs = tokenizer.encode_plus(text, add_special_tokens=True, return_tensors='tf')
-        inputs_onnx = {k_: v_.numpy() for k_, v_ in inputs.items()}
+        # The tokenizers are generated using transformers 2.5.0, but model_max_length is introduced and needed in 2.9.0.
+        if not hasattr(tokenizer, 'model_max_length'):
+            tokenizer.model_max_length = 1024
```

Comment:
This change is for transformers 2.9.0 which is just released one hour ago before the change, which breaks nightly build.